gsoc/admin.py

```diff
@@ -772,3 +772,15 @@ class GsocYearAdmin(admin.ModelAdmin):
 admin.site.register(GsocYear, GsocYearAdmin)
 
 admin.site.register(SubOrg)
+
+
+class NotAcceptedAdmin(admin.ModelAdmin):
+    list_display = ("email", "user_suborg", "user_role")
+    list_filter = ("user_suborg", "user_role")
+
+    def get_queryset(self, request):
+        not_accepted_users = RegLink.objects.filter(is_used=False)
```

Comment:
Lets change this to look for if the email assiciated with the reglink has been used, as you can send more than one reglink to the same email